source/includes/steps-connect-tableau-mongosql-plugin.yaml

```diff
@@ -1,80 +1,163 @@
+       - Save the library file ``mongosql_auth.so`` to the 
+         ``/usr/local/mysql/lib/plugin/`` directory.
+
+         If this directory does not exist, create it:
+
+         .. class:: copyable-code
+         .. code-block:: sh
+
+            sudo mkdir -p /usr/local/mysql/lib/plugin -m 755
+            sudo chown -R {user}:admin /usr/local/mysql
+            tar -xvf /Users/{user}/Downloads/mongosql-auth-osx-x86_64-v{version}.tgz
+            cp /Users/{user}/Downloads/mongosql-auth-osx-x86_64/lib/mongosql_auth.so ./
+
+         .. note:: 
+
+            This example assumes you downloaded the ``mongosql_auth``
+            plugin to the ``/Users/{user}/Downloads/`` directory.
+
+            The plugin does not require MySQL or the MySQL ODBC
+            Connector to be installed.
+
+     * - Windows
+       - Run the Windows installer (``.msi``).
+
 ---
-title: Create a ``my.cnf`` file.
+title: Create a ``my.cnf`` file. (macOS only)
 ref: create-my-cnf
 level: 4
-pre: |
-  OS X
-    Create a text file with the following contents:
+content: |
+
+  a. Create a text file with the following contents:
+
+     .. class:: copyable-code
 
-    .. code-block:: sh
+     .. code-block:: ini
+ 
+        [client]
+        default-auth=mongosql_auth
 
-       [client]
-       default-auth=mongosql_auth
+  b. Save the file to ``/etc/my.cnf``.
 
-    Name the file ``my.cnf`` and save it in the directory ``/etc``.
 ---
-title: Create a ``my.tdc`` file.
-ref: create-my-tdc
+title: Create a Tableau Datasource Connection file.
+ref: create-tdc
 level: 4
-pre: |
-  OS X
-    Create a text file with the following contents:
-
-    .. code-block:: sh
-
-       <?xml version='1.0' encoding='utf-8' ?>
-       <connection-customization class='mongodb' version='7.7' enabled='true'>
-           <vendor name='mongodb' />
-           <driver name='mongodb' />
-           <customizations>
-             <customization
-               name='odbc-connect-string-extras'
-               value='USE_MYCNF=1' />
-             </customizations>
-       </connection-customization>
-
-    Name the file ``my.tdc`` and save in Tableau's ``Datasources``
-    directory. By default, Tableau creates a directory called ``My
-    Tableau Repository`` in your OS X user's ``Documents`` directory,
-    and the ``Datasources`` directory can be found there. Your
-    ``Datasources`` directory may be elsewhere, depending on your
-    system configuration. Place the ``my.tdc`` directory in the
-    repository directory you use while connecting with Tableau.
+content: |
+
+  a. Create a text file with the following contents:
+
+     .. list-table::
+        :widths: 20 80
+        :stub-columns: 1
+        :header-rows: 1
+
+        * - Platform
+          - TDC File Contents
+
+        * - macOS
+          - 
+
+            .. class:: copyable-code
+            .. literalinclude:: /includes/tdc/mongodb-mac.tdc
+               :language: xml
+               :emphasize-lines: 2-4,7-8
```

Comment:
I don't see the value in highlighting certain lines within the .tdc file. They're all equally necessary, even though some are just generic XML tags.